client/stylesheets/_toCat.scss

```diff
@@ -5326,6 +5333,29 @@ On click the images will be displayed at normal size to complete the effect
 
 /* End */
 
+.listingPayment {
+	display: block !important;
+	margin: -75px -100px 0 0;
+}
+
+.listingPayment li {
+	display: inline-block;
+	margin: 0;
+}
+
+.listingPayment li i {
+	font-size: 1.6rem;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered color, font-size